build/integrationtests/README

```diff
@@ -0,0 +1,5 @@
+This Dockerfile and accompanying files are designed to create a Docker image to use for testing LSST software.
+
+It creates an image based on the AlmaLinux base image, and installs Python3.8, Robot-Framework, the SQRE-provided lsst_efd_client package, and other test related dependencies.
```

Comment:
Um, we shouldn't be running Python 3.8. What is the limitation?

Reply:
Fixed.